exotica_core/src/scene.cpp

```diff
@@ -783,6 +783,7 @@ void Scene::UpdateSceneFrames()
 
             std::string collision_element_name = links[i]->getName() + "_collision_" + std::to_string(j);
             std::shared_ptr<KinematicElement> element = kinematica_.AddElement(collision_element_name, trans, links[i]->getName(), links[i]->getShapes()[j]);
+            element->is_robot_link = true;
```

Comment:
Is this necessary? In the collision scene, we always check the `closest_robot_link.lock()` on the `weak_ptr` to check if it is attached to a robot link?

Reply:
Attached objects were always handled via the `closest_robot_link` instead of setting it to `is_robot_link` itself